test/addons/volsync/test

```diff
@@ -238,15 +249,28 @@ os.chdir(os.path.dirname(__file__))
 cluster1, cluster2 = sys.argv[1:]
 
 setup_application(cluster1, "file")
+setup_application(cluster1, "block")
 setup_replication_destination(cluster2, "file")
+setup_replication_destination(cluster2, "block")
 
 wait_for_application(cluster1, "file")
+wait_for_application(cluster1, "block")
 wait_for_replication_destination(cluster2, "file")
+wait_for_replication_destination(cluster2, "block")
 
 setup_replication_secret(cluster1, cluster2, "file")
+setup_replication_secret(cluster1, cluster2, "block")
 setup_replication_service(cluster1, cluster2, "file")
+setup_replication_service(cluster1, cluster2, "block")
+wait_for_replication_service(cluster1, cluster2, "file")
+wait_for_replication_service(cluster1, cluster2, "block")
 
-run_replication(cluster1, "file")
+start_replication(cluster1, "file")
+start_replication(cluster1, "block")
+wait_for_replication(cluster1, "file")
+wait_for_replication(cluster1, "block")
 
 teardown(cluster1, cluster2, "file")
+teardown(cluster1, cluster2, "block")
 wait_for_deletion(cluster1, cluster2, "file")
+wait_for_deletion(cluster1, cluster2, "block")
```

Comment:
The test becomes too complicated now, I think it is time to use executor to run a simple test function with 2 variants instead.